src/net_permissions.cpp

```diff
@@ -9,7 +9,7 @@
 #include <util/translation.h>
 
 // The parse the following format "perm1,perm2@xxxxxx"
-bool TryParsePermissionFlags(const std::string str, NetPermissionFlags& output, size_t& readen, std::string& error)
+static bool TryParsePermissionFlags(const std::string str, NetPermissionFlags& output, size_t& readen, bilingual_str& error)
```

Comment:
nit: It seems using an unnamed namespace is preferable:
- https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Rs-unnamed2
- https://stackoverflow.com/questions/154469/unnamed-anonymous-namespaces-vs-static-functions
- https://stackoverflow.com/questions/4422507/superiority-of-unnamed-namespace-over-static
- https://stackoverflow.com/questions/4977252/why-an-unnamed-namespace-is-a-superior-alternative-to-static